GVFS/GVFS.FunctionalTests.Windows/Windows/Tests/WindowsFileSystemTests.cs

```diff
@@ -1142,6 +1142,7 @@ public void Native_ProjFS_MoveFile_VirtualToOutside()
             ProjFS_MoveFileTest.ProjFS_MoveFile_VirtualToOutside(Path.GetDirectoryName(this.Enlistment.RepoRoot), this.Enlistment.RepoRoot).ShouldEqual(true);
         }
 
+        [Ignore("Disable this test until we can surface native test errors, see #454")]

```

Comment:
Nit: a full URL would be more helpful here, since whoever is reading this is likely to land on it inside of an IDE

Reply:
And I will immediately take that back, since we haven't done it on ANY of our other TODOs or Ignores :-)